jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -875,33 +875,14 @@ private void resetLastPresenceSent()
         @Override
         public void pingFailed()
         {
-            logger.warn("XMPP Ping failed");
+            logger.warn("Ping failed, the XMPP connection needs to reconnect.");
```

Comment:
This logs seems misleading.  Is a failed ping necessarily a disconnection?  Especially since `xmppConnection.isConnected()` may return true below?

Reply:
It needs to reconnect, that's why if it's still connected we disconnect